### PR TITLE
Buffs the powerfist

### PR DESCRIPTION
**Pull request by PopeDaveThe3th**
## What Does This PR Do
Reduces powerfist gas consumption by 5/6ths.

Adds nodrop whenever a gas tank is inserted into the fist, removes nodrop when the tank is removed.
## Why It's Good For The Game

The powerfist is a fairly terrible traitor item, having some of the worst ammo efficiency of any weapon in the game. With a stock oxygen tank, you can use the fist's maximum setting only twice before running dry.

This PR aims to make the fist a bit more competitive as a damage dealer.

Additionally, nodrop was added at the request of Qwertytoforty.

## Testing
- Loaded test server
- Spawned fist
- Tested loading and unloading the fist. (works with both stock oxygen and plasma tanks)
- Beat numerous vox to death
- Fist works as intended

## Changelog
:cl:
tweak: Buffed the powerfist, lowering gas consumption and adding nodrop
/:cl:

### CODE REVIEW

**Comment by hal9000PR** [comment]
what happens when you put a tank in the fist without holding the fist?

File: code/game/objects/items/weapons/powerfist.dm [hunk 1]
```diff
@@ -14,10 +14,9 @@
 	origin_tech = "combat=5;powerstorage=3;syndicate=3"
 	var/click_delay = 1.5
 	var/fisto_setting = 1
-	var/gasperfist = 3
+	var/gasperfist = 0.5 //original consumption mod was 3, for reference
```
Comment (hal9000PR) [inline code/game/objects/items/weapons/powerfist.dm:17]:
```suggestion
	var/gasperfist = 0.5
```
please dont leave comments like this, we have git history for a reason (this var name is also awful, not your fault though)

**Comment by PopeDaveThe3th** [comment]
> what happens when you put a tank in the fist without holding the fist?

It still shows the "locks around your arm" message, but the nodrop is applied as usual and removed if you pop the tank out.

I can change the tank insert message to be a bit more ambiguous if you'd like.

**Review by Sirryan2002 — COMMENTED** [review]
Can you check to see what happens when someone's arm gets blown off while using the powerfist. If the powerfist gets dropped from this, it shouldn't immediately glue itself to someone's hand if they pick it up off the ground.

**Comment by PopeDaveThe3th** [comment]
> Can you check to see what happens when someone's arm gets blown off while using the powerfist. If the powerfist gets dropped from this, it shouldn't immediately glue itself to someone's hand if they pick it up off the ground.

Would it be acceptable if I made it so that you can't pick the fist up while a tank is inserted?

**Comment by Sirryan2002** [comment]
It should remove the NODROP upon a limb being blown off due to the base item `dropped()` proc, however I'm just asking you to test to see if this is actually the case.

**Comment by PopeDaveThe3th** [comment]
> It should remove the NODROP upon a limb being blown off due to the base item `dropped()` proc, however I'm just asking you to test to see if this is actually the case.

Can confirm, blew self up with a grenade, lost left arm, fist dropped even though I was holding it in my right arm. The nodrop is reapplied by extracting and replacing the tank.

I'll chalk this one up lorewise to "emergency failsafe that prevents you from getting the fist permanently stuck if you lose a limb".

**Review by Sirryan2002 — APPROVED** [review]
code looks fine

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**